python/ray/data/_internal/datasource/parquet_datasource.py

```diff
@@ -463,10 +464,10 @@ def read_fragments(
             }
 
         def get_batch_iterable():
-            return fragment.to_batches(
+            pq_file = pq.ParquetFile(fragment.path)
```

Comment:
Please add a comment elaborating why we're switching to `iter_batches` and link upstream issue here